inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -200,6 +210,33 @@ def is_submission(obj, eng):
     return False
 
 
+@with_debug_logging
+def _get_journal_title(obj, eng):
+    """Return the journal title of the publication."""
+    publication = obj.data.get('publication_info')
+    if publication:
+        return publication[0].get('journal_title')
```

Comment:
@michamos this is wrong, right? As there can be more journals? I think I made this mistake in the past too. Also, if there are more journals in `publication_info`, then which one do I consider for the coverage?

Reply:
I don't think you should do a query in `get_journal_coverage`. If this runs after `normalize_journal_title` from #2938, then there is already the `$ref` to the journal and you can use a strategy similar to #2951.
If any journal has full coverage, then the `reject` button should be greyed out.